operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -135,6 +135,11 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=8,displayName="Declarative Configuration",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
 	DeclarativeConfiguration *DeclarativeConfiguration `json:"declarativeConfiguration,omitempty"`
 
+	// Enables the encryption of notifier secrets stored in the Central DB. An encryption key must be
+	// provided in a secret called `central-encryption-key` in the Central namespace.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=9,displayName="Encrypt Notifier Secrets",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
+	EncryptNotifierSecrets *bool `json:"encryptNotifierSecrets,omitempty"`
```

Comment:
@johannes94 Should this be a struct with a single `Enabled` field, to support future extensions?

E.g. I'm thinking that in the future the Operator might have other options such as `AutoGenerateEncryptionKey`.